web/apps/apps.go

```diff
@@ -214,6 +257,36 @@ func installHandler(installerType consts.AppType) echo.HandlerFunc {
 	}
 }
 
+// logsHandler handles all POST /:slug/logs requests and forwards to sent log
+// lines to the server logger.
+func logsHandler(appType consts.AppType) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		inst := middlewares.GetInstance(c)
+		slug := c.Param("slug")
+		if err := middlewares.AllowWholeType(c, permission.POST, consts.AppLogs); err != nil {
+			return err
+		}
+
+		body, err := ioutil.ReadAll(c.Request().Body)
+		if err != nil {
+			return err
+		}
+
+		var logs []AppLog
+		if err := json.Unmarshal(body, &logs); err != nil {
+			return err
+		}
+
+		logger := logger.WithDomain(inst.Domain).WithNamespace(slug)
```

Comment:
The server-side konnectors have their logs with the slug in a `slug` field. We can use `konnectors` or `apps` for the namespace.